### ISSUE TITLE
Pertinence de hasSubActivity ?

### ISSUE THREAD

**Issue by srosset81** [assemblee-virtuelle/pair#137]
Etant donné que `hasPart` / `partOf` peut s'appliquer à n'importe quel sujet, pourquoi est-il nécessaire d'avoir des prédicats `hasSubActivity` / `subActivityOf` ?

S'il s'agit de dire qu'un projet fait partie d'un projet plus large, il me semble que `hasPart` / `partOf` serait adéquat.

**Comment by tfrancart** [comment]
Correct, je l'avais gardé pour ne pas perdre les utilisateurs; le petit avantage est que le range est restreint : hasSubActivity prend ses valeurs seulement dans Activity.
hasSubActivity est définie comme une sous-propriété de hasPart.

**Comment by tfrancart** [comment]
Est-ce que tu préfères qu'on enlève hasSubActivity ?

**Comment by srosset81** [comment]
Je pense qu'on peut s'en passer oui !

**Comment by tfrancart** [comment]
@srosset81 : je supprime carrément la propriété ? ou tu veux que je la garde en la marquant owl:deprecated ?
Je préfère supprimer carrément

**Comment by simonLouvet** [comment]
pas utilisé non plus
